superset-frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -20,7 +20,7 @@ import React from 'react';
 import { styled } from '@superset-ui/core';
 import { CheckboxChecked, CheckboxUnchecked } from 'src/components/Checkbox';
 
-interface CheckboxProps {
+export interface CheckboxProps {
```

Comment:
Bycatch: fix this missing export. For some reason, this is causing Cypress test on SQL tabs to fail locally.